test/utils.spec.js

```diff
@@ -262,15 +285,31 @@ describe('utils', function() {
 });
 
 /* eslint-disable no-console */
-function handleConsoleWarns(doWithShimmedConsoleWarn, onWarn) {
-  // Shim `console.warn`
-  var _warn = console.warn;
+function handleWarnings(doWithShimmedConsoleWarn, onWarn) {
+  /* eslint-disable no-console */
```

Comment:
Another small one, but I think this `eslint-disable` might have been duplicated. Can we just have one of these? (Between the ones on lines 287 and 289.)